src/components/Upload/UploadFile.vue

```diff
@@ -1,30 +1,52 @@
 <template>
-	<div
-		class="card-body flex justify-center border-3 border-gray-400 border-dashed 40vh"
-	>
-		<div class="flex flex-col justify-center items-center">
-			<i class="fa fa-cloud-upload-alt my-8 text-gray-400 text-6xl"></i>
-			<h3 class="my-1 text-md leading-5 text-gray-800">
-				Pretiahnite súbory sem
-			</h3>
-			<p class="my-1 text-md text-gray-600">alebo</p>
-			<div class="mx-auto my-4">
-				<label
-					class="flex text-white rounded bg-primary-500 hover:bg-primary-400 px-4 py-2 cursor-pointer transition duration-500 ease select-none focus:outline-none"
-				>
-					Vybrať súbory
-					<input
-						id="file"
-						ref="file"
-						type="file"
-						multiple
-						class="hidden"
-						@change="handleFileUpload"
-					/>
-				</label>
-				<button v-on:click="submitFile">Submit</button>
+	<div class="space-y-6">
+		<div class="card-body border-3 border-gray-400 border-dashed 40vh">
+			<div class="card-body flex flex-col justify-center bg-primary-100 hover:bg-white">
+				<form ref="fileform" class="cursor-pointer">
+					<div class="flex flex-col justify-center items-center space-y-2 py-8">
+						<i class="fa fa-cloud-upload-alt text-gray-500 text-6xl"></i>
+						<h2 class="text-xl font-semibold leading-6 text-gray-800">
+							Pretiahnite súbory sem
+						</h2>
+						<p class="text-md text-gray-600">alebo</p>
+						<div class="mx-auto my-4">
+							<label
+								class="flex text-white rounded shadow-md bg-primary-500 hover:bg-primary-400 px-6 py-2 
+								cursor-pointer transition duration-500 ease select-none focus:outline-none"
+							>
+								Vybrať súbory
+								<input
+									id="files"
+									ref="files"
+									type="file"
+									multiple
+									class="hidden"
+									@change="handleFileUpload"
+								/>
+							</label>
+						</div>				
+					</div>
+				</form>
 			</div>
 		</div>
+		
+		<div v-show="this.$parent.files.length > 0" class="overflow-auto h-40 bg-primary-100 shadow-md">
+			<div v-for="(file, key) in this.$parent.files" 
+				:key="key" 
+				class="flex justify-between items-center px-4 my-2 border-b-3 border-gray-400"
+			>				
+				<div class="flex items-center space-x-4">
+					<i class="fas fa-file text-gray-500 text-3xl hover:text-gray-600"></i>
```

Comment:
Why are we using `hover` class here? The icon is not clickable, I don't think hover should be here.